ivy/src/main/scala/sbt/IvyScala.scala

```diff
@@ -47,17 +50,39 @@ private object IvyScala {
   /** Performs checks/adds filters on Scala dependencies (if enabled in IvyScala). */
   def checkModule(module: DefaultModuleDescriptor, conf: String, log: Logger)(check: IvyScala): Unit = {
     if (check.checkExplicit)
-      checkDependencies(module, check.scalaBinaryVersion, check.configurations, log)
+      checkDependencies(module, check.scalaOrganization, check.scalaBinaryVersion, check.configurations, log)
     if (check.filterImplicit)
       excludeScalaJars(module, check.configurations)
-    if (check.overrideScalaVersion)
-      overrideScalaVersion(module, check.scalaFullVersion)
+    if (true || check.overrideScalaVersion)
```

Comment:
> If an attribute of `UpdateOptions`, how would that be made available to `IvyScala`?

Maybe we can use this `overrideScalaVersion`, which is where you're hooking your code anyway.
I'm ok with flipping the default value of this flag to `true`.